examples/acquiring_data/downloading_cutouts.py

```diff
@@ -74,13 +74,15 @@
 
 #####################################################
 # Submit the export request and download the data.
+
 files = Fido.fetch(query)
 files.sort()
 
 #####################################################
 # Now that we've downloaded the files, we can create
 # a `~sunpy.map.MapSequence` from them and animate
 # them.
+
 sequence = sunpy.map.Map(files, sequence=True)
 # Make sure the colorbar limits are the same for each image
 for each_map in sequence:
```

Comment:
Since we're here, can you remove this for loop and just put the `norm=` directly in the `sequence.plot` call? When I first wrote this example, that didn't work.

Reply:
Give me a minute.

Reply:
Removed.